src/include/mallocMC/creationPolicies/Scatter_impl.hpp

```diff
@@ -623,6 +617,7 @@ namespace ScatterKernelDetail{
 #else
           unsigned int __mask = __ballot(1),
 #endif
+            // TODO(bgruber): why the leading __? those names are reserved
```

Comment:
The naming is a leftover from the original ScatterAlloc implementation. Should be no harm to name them properly, yep!

Reply:
done